src/lib/gundb/__tests__/UserStorage.js

```diff
@@ -343,6 +343,7 @@ describe('UserStorage', () => {
   })
 
   it('has the welcome event already set', async () => {
+    await userStorage.updateFeedEvent(welcomeMessage)
```

Comment:
You need to remove this line and fix the ID for the welcomeMessage card
src/lib/gundb/UserStorageClass.js 109

Reply:
Done